### ISSUE TITLE
[App]: make test failed about wandb@0.13.5 on centos8_aarch64

### ISSUE THREAD

**Issue by Tom-python0121** [wandb/wandb#4578]
### Current Behavior

hello,I meet a problem:make test failed about wandb@0.13.5 on centos8_aarch64

### Expected Behavior

the tests can passed.

### Steps to reproduce the issue

```console
[root@centos spack-src]# make -j126 test
tox -e "pyupgrade,black,mypy,flake8,docstrings"
GLOB sdist-make: /home/stage/root/spack-stage-py-wandb-0.13.5-znwfiap4aaawa45zetk52txtvmzjfbrh/spack-src/setup.py
pyupgrade inst-nodeps: /home/stage/root/spack-stage-py-wandb-0.13.5-znwfiap4aaawa45zetk52txtvmzjfbrh/spack-src/.tox/.tmp/package/1/wandb-0.13.5.zip
WARNING: Discarding $PYTHONPATH from environment, to override specify PYTHONPATH in 'passenv' in your configuration.
pyupgrade installed: aiofiles==0.5.0,aiohttp==3.6.2,amqp==5.0.1,apache-beam==2.24.0,asgiref==3.2.7,async-timeout==3.0.1,attrs==19.3.0,avro-python3==1.10.0,Babel==2.7.0,billiard==3.6.3.0,botocore==1.13.44,bottle==0.12.18,brotlipy==0.7.0,celery==5.0.0,certifi==2020.6.20,cffi==1.14.3,chalice==1.20.0,chardet==3.0.4,charset-normalizer==2.1.1,click==7.1.2,click-didyoumean==0.0.3,click-repl==0.1.6,configparser==3.8.1,crcmod==1.7,Cython==0.29.21,dill==0.3.1.dev0,Django==3.0.5,docker-pycreds==0.4.0,docopt==0.6.2,docutils==0.15.2,falcon==3.0.0a2,fastavro==0.24.2,Flask==1.1.2,future==0.18.2,gitdb==4.0.5,GitPython==3.1.12,gql==3.0.0a1,graphql-core==3.1.2,grpcio==1.32.0,h11==0.9.0,h2==4.0.0,hdfs==2.1.0,hpack==4.0.0,hstspreload==2020.9.23,httpcore==0.11.0,httplib2==0.13.1,httptools==0.1.1,httpx==0.11.1,hyperframe==6.0.0,idna==2.8,importlib-metadata==2.0.0,itsdangerous==1.1.0,Jinja2==3.0.1,jmespath==0.9.4,kombu==5.0.2,lz4==3.1.0,MarkupSafe==2.0.1,mock==2.0.0,more-itertools==7.2.0,multidict==4.7.6,mypy-extensions==0.4.3,numpy==1.20.1,nvidia-ml-py3==7.352.0,oauth2client==4.1.3,pathtools==0.1.2,promise==2.3,prompt-toolkit==3.0.16,protobuf==3.12.2,psutil==5.7.2,pure-eval==0.2.2,py-snappy==0.1.0a1,py4j==0.10.9,pyarrow==0.17.1,pyasn1==0.4.6,pyasn1-modules==0.2.6,pycparser==2.20,pydot==1.4.1,pymongo==3.9.0,pyparsing==2.4.7,pyspark==3.0.1,pytz==2020.1,pyupgrade==2.34.0,PyYAML==5.3.1,redis==3.5.3,requests==2.24.0,rfc3986==1.4.0,rq==1.5.2,rsa==4.0,sanic==20.6.3,sentry-sdk==0.17.6,setproctitle==1.3.2,shortuuid==1.0.1,six==1.15.0,smmap==3.0.4,sniffio==1.1.0,SQLAlchemy==1.3.19,sqlparse==0.4.1,subprocess32==3.5.4,tokenize-rt==5.0.0,tornado==6.1,ujson==1.35,urllib3==1.25.6,uvloop==0.14.0,vine==5.0.0,wandb==0.13.5,watchdog==2.1.6,wcwidth==0.1.7,websockets==8.1,Werkzeug==0.16.0,yarl==1.4.2,zipp==0.6.0
pyupgrade run-test-pre: PYTHONHASHSEED='2893514355'
pyupgrade run-test: commands[0] | sh -c 'python tools/locate-py-files.py | xargs pyupgrade --py36-plus --keep-mock'
black installed: aiofiles==0.5.0,aiohttp==3.6.2,amqp==5.0.1,apache-beam==2.24.0,asgiref==3.2.7,async-timeout==3.0.1,attrs==19.3.0,avro-python3==1.10.0,Babel==2.7.0,billiard==3.6.3.0,black==22.3.0,botocore==1.13.44,bottle==0.12.18,brotlipy==0.7.0,celery==5.0.0,certifi==2020.6.20,cffi==1.14.3,chalice==1.20.0,chardet==3.0.4,click==7.1.2,click-didyoumean==0.0.3,click-repl==0.1.6,configparser==3.8.1,crcmod==1.7,Cython==0.29.21,dill==0.3.1.dev0,Django==3.0.5,docker-pycreds==0.4.0,docopt==0.6.2,docutils==0.15.2,falcon==3.0.0a2,fastavro==0.24.2,Flask==1.1.2,future==0.18.2,gitdb==4.0.5,GitPython==3.1.12,gql==3.0.0a1,graphql-core==3.1.2,grpcio==1.32.0,h11==0.9.0,h2==4.0.0,hdfs==2.1.0,hpack==4.0.0,hstspreload==2020.9.23,httpcore==0.11.0,httplib2==0.13.1,httptools==0.1.1,httpx==0.11.1,hyperframe==6.0.0,idna==2.8,importlib-metadata==2.0.0,itsdangerous==1.1.0,Jinja2==3.0.1,jmespath==0.9.4,kombu==5.0.2,lz4==3.1.0,MarkupSafe==2.0.1,mock==2.0.0,more-itertools==7.2.0,multidict==4.7.6,mypy-extensions==0.4.3,numpy==1.20.1,nvidia-ml-py3==7.352.0,oauth2client==4.1.3,pathspec==0.10.2,pathtools==0.1.2,platformdirs==2.5.4,promise==2.3,prompt-toolkit==3.0.16,protobuf==3.12.2,psutil==5.7.2,pure-eval==0.2.2,py-snappy==0.1.0a1,py4j==0.10.9,pyarrow==0.17.1,pyasn1==0.4.6,pyasn1-modules==0.2.6,pycparser==2.20,pydot==1.4.1,pymongo==3.9.0,pyparsing==2.4.7,pyspark==3.0.1,pytz==2020.1,PyYAML==5.3.1,redis==3.5.3,requests==2.24.0,rfc3986==1.4.0,rq==1.5.2,rsa==4.0,sanic==20.6.3,sentry-sdk==0.17.6,shortuuid==1.0.1,six==1.15.0,smmap==3.0.4,sniffio==1.1.0,SQLAlchemy==1.3.19,sqlparse==0.4.1,subprocess32==3.5.4,tomli==2.0.1,tornado==6.1,typing-extensions==4.4.0,ujson==1.35,urllib3==1.25.6,uvloop==0.14.0,vine==5.0.0,wandb==0.13.5,watchdog==2.1.6,wcwidth==0.1.7,websockets==8.1,Werkzeug==0.16.0,yarl==1.4.2,zipp==0.6.0
black run-test-pre: PYTHONHASHSEED='2893514355'
black run-test: commands[0] | black --check wandb/ tests/ tools/
Skipping .ipynb files as Jupyter dependencies are not installed.
You can fix this by running ``pip install black[jupyter]``
All done! ✨ 🍰 ✨
623 files would be left unchanged.
mypy installed: aiofiles==0.5.0,aiohttp==3.6.2,amqp==5.0.1,apache-beam==2.24.0,asgiref==3.2.7,async-timeout==3.0.1,attrs==19.3.0,avro-python3==1.10.0,Babel==2.7.0,billiard==3.6.3.0,botocore==1.13.44,bottle==0.12.18,brotlipy==0.7.0,celery==5.0.0,certifi==2020.6.20,cffi==1.14.3,chalice==1.20.0,chardet==3.0.4,click==7.1.2,click-didyoumean==0.0.3,click-repl==0.1.6,configparser==3.8.1,crcmod==1.7,Cython==0.29.21,dill==0.3.1.dev0,Django==3.0.5,docker-pycreds==0.4.0,docopt==0.6.2,docutils==0.15.2,falcon==3.0.0a2,fastavro==0.24.2,Flask==1.1.2,future==0.18.2,gitdb==4.0.5,GitPython==3.1.12,gql==3.0.0a1,graphql-core==3.1.2,grpcio==1.32.0,h11==0.9.0,h2==4.0.0,hdfs==2.1.0,hpack==4.0.0,hstspreload==2020.9.23,httpcore==0.11.0,httplib2==0.13.1,httptools==0.1.1,httpx==0.11.1,hyperframe==6.0.0,idna==2.8,importlib-metadata==2.0.0,itsdangerous==1.1.0,Jinja2==3.0.1,jmespath==0.9.4,kombu==5.0.2,lxml==4.9.1,lz4==3.1.0,MarkupSafe==2.0.1,mock==2.0.0,more-itertools==7.2.0,multidict==4.7.6,mypy==0.971,mypy-extensions==0.4.3,numpy==1.20.1,nvidia-ml-py3==7.352.0,oauth2client==4.1.3,pathtools==0.1.2,promise==2.3,prompt-toolkit==3.0.16,protobuf==3.12.2,psutil==5.7.2,pure-eval==0.2.2,py-snappy==0.1.0a1,py4j==0.10.9,pyarrow==0.17.1,pyasn1==0.4.6,pyasn1-modules==0.2.6,pycparser==2.20,pydot==1.4.1,pymongo==3.9.0,pyparsing==2.4.7,pyspark==3.0.1,pytz==2020.1,PyYAML==5.3.1,redis==3.5.3,requests==2.24.0,rfc3986==1.4.0,rq==1.5.2,rsa==4.0,sanic==20.6.3,sentry-sdk==0.17.6,shortuuid==1.0.1,six==1.15.0,smmap==3.0.4,sniffio==1.1.0,SQLAlchemy==1.3.19,sqlparse==0.4.1,subprocess32==3.5.4,tomli==2.0.1,tornado==6.1,types-click==7.1.8,types-protobuf==4.21.0.2,types-PyYAML==6.0.12.2,types-requests==2.28.11.5,types-setuptools==65.6.0.1,types-six==1.16.21.4,types-urllib3==1.26.25.4,typing-extensions==4.4.0,ujson==1.35,urllib3==1.25.6,uvloop==0.14.0,vine==5.0.0,wandb==0.13.5,watchdog==2.1.6,wcwidth==0.1.7,websockets==8.1,Werkzeug==0.16.0,yarl==1.4.2,zipp==0.6.0
mypy run-test-pre: PYTHONHASHSEED='2893514355'
mypy run-test: commands[0] | mypy --install-types --non-interactive --show-error-codes --config-file /home/stage/root/spack-stage-py-wandb-0.13.5-znwfiap4aaawa45zetk52txtvmzjfbrh/spack-src/mypy.ini -p wandb --html-report mypy-results/ --cobertura-xml-report mypy-results/ --lineprecision-report mypy-results/
Generated Cobertura report: /home/stage/root/spack-stage-py-wandb-0.13.5-znwfiap4aaawa45zetk52txtvmzjfbrh/spack-src/mypy-results/cobertura.xml
Generated HTML report (via XSLT): /home/stage/root/spack-stage-py-wandb-0.13.5-znwfiap4aaawa45zetk52txtvmzjfbrh/spack-src/mypy-results/index.html
Installing missing stub packages:
/home/stage/root/spack-stage-py-wandb-0.13.5-znwfiap4aaawa45zetk52txtvmzjfbrh/spack-src/.tox/mypy/bin/python -m pip install types-protobuf

Looking in indexes: http://mirrors.tools.huawei.com/pypi/simple/
Requirement already satisfied: types-protobuf in ./.tox/mypy/lib/python3.8/site-packages (4.21.0.2)

[notice] A new release of pip available: 22.2.2 -> 22.3.1
[notice] To update, run: pip install --upgrade pip

Generated Cobertura report: /home/stage/root/spack-stage-py-wandb-0.13.5-znwfiap4aaawa45zetk52txtvmzjfbrh/spack-src/mypy-results/cobertura.xml
Generated HTML report (via XSLT): /home/stage/root/spack-stage-py-wandb-0.13.5-znwfiap4aaawa45zetk52txtvmzjfbrh/spack-src/mypy-results/index.html
wandb/sdk/lib/proto_util.py:9: error: Library stubs not installed for "google.protobuf.internal.containers" (or incompatible with Python 3.8)  [import]
wandb/sdk/lib/proto_util.py:9: note: Hint: "python3 -m pip install types-protobuf"
wandb/sdk/lib/proto_util.py:9: note: (or run "mypy --install-types" to install all missing stub packages)
wandb/sdk/lib/proto_util.py:9: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
wandb/sdk/lib/proto_util.py:13: error: Library stubs not installed for "google.protobuf.message" (or incompatible with Python 3.8)  [import]
wandb/sdk/lib/proto_util.py:93: error: Library stubs not installed for "google.protobuf.json_format" (or incompatible with Python 3.8)  [import]
wandb/sdk/lib/proto_util.py:95: error: Returning Any from function declared to return "Dict[str, Any]"  [no-any-return]
wandb/sdk/service/service_base.py:17: error: Library stubs not installed for "google.protobuf.internal.containers" (or incompatible with Python 3.8)  [import]
Found 5 errors in 2 files (checked 311 source files)
ERROR: InvocationError for command /home/stage/root/spack-stage-py-wandb-0.13.5-znwfiap4aaawa45zetk52txtvmzjfbrh/spack-src/.tox/mypy/bin/mypy --install-types --non-interactive --show-error-codes --config-file mypy.ini -p wandb --html-report mypy-results/ --cobertura-xml-report mypy-results/ --lineprecision-report mypy-results/ (exited with code 1)
flake8 installed: aiofiles==0.5.0,aiohttp==3.6.2,amqp==5.0.1,apache-beam==2.24.0,asgiref==3.2.7,async-timeout==3.0.1,attrs==19.3.0,avro-python3==1.10.0,Babel==2.7.0,billiard==3.6.3.0,botocore==1.13.44,bottle==0.12.18,brotlipy==0.7.0,celery==5.0.0,certifi==2020.6.20,cffi==1.14.3,chalice==1.20.0,chardet==3.0.4,click==7.1.2,click-didyoumean==0.0.3,click-repl==0.1.6,configparser==3.8.1,crcmod==1.7,Cython==0.29.21,dill==0.3.1.dev0,Django==3.0.5,docker-pycreds==0.4.0,docopt==0.6.2,docutils==0.15.2,falcon==3.0.0a2,fastavro==0.24.2,flake8==6.0.0,flake8-bugbear==22.10.27,flake8-fixme==1.1.1,flake8-typing-imports==1.14.0,Flask==1.1.2,future==0.18.2,gitdb==4.0.5,GitPython==3.1.12,gql==3.0.0a1,graphql-core==3.1.2,grpcio==1.32.0,h11==0.9.0,h2==4.0.0,hdfs==2.1.0,hpack==4.0.0,hstspreload==2020.9.23,httpcore==0.11.0,httplib2==0.13.1,httptools==0.1.1,httpx==0.11.1,hyperframe==6.0.0,idna==2.8,importlib-metadata==2.0.0,itsdangerous==1.1.0,Jinja2==3.0.1,jmespath==0.9.4,kombu==5.0.2,lz4==3.1.0,MarkupSafe==2.0.1,mccabe==0.7.0,mock==2.0.0,more-itertools==7.2.0,multidict==4.7.6,mypy-extensions==0.4.3,numpy==1.20.1,nvidia-ml-py3==7.352.0,oauth2client==4.1.3,pathtools==0.1.2,pep8-naming==0.13.2,promise==2.3,prompt-toolkit==3.0.16,protobuf==3.12.2,psutil==5.7.2,pure-eval==0.2.2,py-snappy==0.1.0a1,py4j==0.10.9,pyarrow==0.17.1,pyasn1==0.4.6,pyasn1-modules==0.2.6,pycodestyle==2.10.0,pycparser==2.20,pydot==1.4.1,pyflakes==3.0.1,pymongo==3.9.0,pyparsing==2.4.7,pyspark==3.0.1,pytz==2020.1,PyYAML==5.3.1,redis==3.5.3,requests==2.24.0,rfc3986==1.4.0,rq==1.5.2,rsa==4.0,sanic==20.6.3,sentry-sdk==0.17.6,shortuuid==1.0.1,six==1.15.0,smmap==3.0.4,sniffio==1.1.0,SQLAlchemy==1.3.19,sqlparse==0.4.1,subprocess32==3.5.4,tornado==6.1,ujson==1.35,urllib3==1.25.6,uvloop==0.14.0,vine==5.0.0,wandb==0.13.5,watchdog==2.1.6,wcwidth==0.1.7,websockets==8.1,Werkzeug==0.16.0,yarl==1.4.2,zipp==0.6.0
flake8 run-test-pre: PYTHONHASHSEED='2893514355'
flake8 run-test: commands[0] | flake8 --append-config=/home/stage/root/spack-stage-py-wandb-0.13.5-znwfiap4aaawa45zetk52txtvmzjfbrh/spack-src/.flake8-base --color=always
docstrings installed: aiofiles==0.5.0,aiohttp==3.6.2,amqp==5.0.1,apache-beam==2.24.0,asgiref==3.2.7,async-timeout==3.0.1,attrs==19.3.0,avro-python3==1.10.0,Babel==2.7.0,billiard==3.6.3.0,botocore==1.13.44,bottle==0.12.18,brotlipy==0.7.0,celery==5.0.0,certifi==2020.6.20,cffi==1.14.3,chalice==1.20.0,chardet==3.0.4,click==7.1.2,click-didyoumean==0.0.3,click-repl==0.1.6,configparser==3.8.1,crcmod==1.7,Cython==0.29.21,dill==0.3.1.dev0,Django==3.0.5,docker-pycreds==0.4.0,docopt==0.6.2,docutils==0.15.2,falcon==3.0.0a2,fastavro==0.24.2,flake8==6.0.0,flake8-docstrings==1.6.0,Flask==1.1.2,future==0.18.2,gitdb==4.0.5,GitPython==3.1.12,gql==3.0.0a1,graphql-core==3.1.2,grpcio==1.32.0,h11==0.9.0,h2==4.0.0,hdfs==2.1.0,hpack==4.0.0,hstspreload==2020.9.23,httpcore==0.11.0,httplib2==0.13.1,httptools==0.1.1,httpx==0.11.1,hyperframe==6.0.0,idna==2.8,importlib-metadata==2.0.0,itsdangerous==1.1.0,Jinja2==3.0.1,jmespath==0.9.4,kombu==5.0.2,lz4==3.1.0,MarkupSafe==2.0.1,mccabe==0.7.0,mock==2.0.0,more-itertools==7.2.0,multidict==4.7.6,mypy-extensions==0.4.3,numpy==1.20.1,nvidia-ml-py3==7.352.0,oauth2client==4.1.3,pathtools==0.1.2,promise==2.3,prompt-toolkit==3.0.16,protobuf==3.12.2,psutil==5.7.2,pure-eval==0.2.2,py-snappy==0.1.0a1,py4j==0.10.9,pyarrow==0.17.1,pyasn1==0.4.6,pyasn1-modules==0.2.6,pycodestyle==2.10.0,pycparser==2.20,pydocstyle==6.1.1,pydot==1.4.1,pyflakes==3.0.1,pymongo==3.9.0,pyparsing==2.4.7,pyspark==3.0.1,pytz==2020.1,PyYAML==5.3.1,redis==3.5.3,requests==2.24.0,rfc3986==1.4.0,rq==1.5.2,rsa==4.0,sanic==20.6.3,sentry-sdk==0.17.6,shortuuid==1.0.1,six==1.15.0,smmap==3.0.4,sniffio==1.1.0,snowballstemmer==2.2.0,SQLAlchemy==1.3.19,sqlparse==0.4.1,subprocess32==3.5.4,tornado==6.1,ujson==1.35,urllib3==1.25.6,uvloop==0.14.0,vine==5.0.0,wandb==0.13.5,watchdog==2.1.6,wcwidth==0.1.7,websockets==8.1,Werkzeug==0.16.0,yarl==1.4.2,zipp==0.6.0
docstrings run-test-pre: PYTHONHASHSEED='2893514355'
docstrings run-test: commands[0] | flake8 --select D --append-config=/home/stage/root/spack-stage-py-wandb-0.13.5-znwfiap4aaawa45zetk52txtvmzjfbrh/spack-src/.flake8-docstrings
______________________________________________________________________________________________ summary ______________________________________________________________________________________________
  pyupgrade: commands succeeded
  black: commands succeeded
ERROR:   mypy: commands failed
  flake8: commands succeeded
  docstrings: commands succeeded
make: *** [Makefile:54: test] Error 1
```
Can you tell me how to solve this problem?

### Environment

OS:Linux centos 4.18.0-193.el8.aarch64 #1 SMP Fri May 8 11:05:12 UTC 2020 aarch64 aarch64 aarch64 GNU/Linux

Browsers:chrom

Version:0.13.5

**Comment by ramit-wandb** [comment]
Hi @Tom-python0121,

Looks like `protobuf` might not be installed correctly on your system - could you try uninstalling and reinstalling protobuf?

**Comment by Tom-python0121** [comment]
@ramit-wandb This bug has been fixed in py-wandb@0.13.6.
### Steps to reproduce the issue

```console
[root@bigdata spack-src]# tox -e mypy
mypy create: /home/stage/root/spack-stage-py-wandb-0.13.6-pjmssa244wsn6ybym6tqvxdmegj7h6u6/spack-src/.tox/mypy
mypy installdeps: types-click==7.1.8, mypy==0.991, lxml, grpcio
WARNING: Discarding $PYTHONPATH from environment, to override specify PYTHONPATH in 'passenv' in your configuration.
mypy installed: grpcio==1.51.1,lxml==4.9.1,mypy==0.991,mypy-extensions==0.4.3,tomli==2.0.1,types-click==7.1.8,typing-extensions==4.4.0
mypy run-test-pre: PYTHONHASHSEED='3574102682'
mypy run-test: commands[0] | mypy --install-types --non-interactive --show-error-codes --config-file /home/stage/root/spack-stage-py-wandb-0.13.6-pjmssa244wsn6ybym6tqvxdmegj7h6u6/spack-src/mypy.ini -p wandb --html-report mypy-results/ --cobertura-xml-report mypy-results/ --lineprecision-report mypy-results/
Generated Cobertura report: /home/stage/root/spack-stage-py-wandb-0.13.6-pjmssa244wsn6ybym6tqvxdmegj7h6u6/spack-src/mypy-results/cobertura.xml
Generated HTML report (via XSLT): /home/stage/root/spack-stage-py-wandb-0.13.6-pjmssa244wsn6ybym6tqvxdmegj7h6u6/spack-src/mypy-results/index.html
Installing missing stub packages:
/home/stage/root/spack-stage-py-wandb-0.13.6-pjmssa244wsn6ybym6tqvxdmegj7h6u6/spack-src/.tox/mypy/bin/python -m pip install types-Pillow types-PyYAML types-google-cloud-ndb types-protobuf types-requests types-setuptools types-six types-urllib3

Looking in indexes: http://mirrors.tools.huawei.com/pypi/simple/
Collecting types-Pillow
  Downloading http://mirrors.tools.huawei.com/pypi/packages/58/19/08eb2937dd8ed441166bb27d69dd87b19f1d51c49b137b99e6f293e9d054/types_Pillow-9.3.0.4-py3-none-any.whl (44 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 44.2/44.2 kB 484.5 kB/s eta 0:00:00
Collecting types-PyYAML
  Downloading http://mirrors.tools.huawei.com/pypi/packages/13/c2/aff0c123a4a8e4b360a52bb62019949fa691d02ca0fda1ac5010171d9d42/types_PyYAML-6.0.12.2-py3-none-any.whl (14 kB)
Collecting types-google-cloud-ndb
  Downloading http://mirrors.tools.huawei.com/pypi/packages/fe/15/dd778fd716a90582e93b7a7e8b38e2c03d515dbaa6ed34a576596a79d09a/types_google_cloud_ndb-2.0.0.0-py3-none-any.whl (16 kB)
Collecting types-protobuf
  Downloading http://mirrors.tools.huawei.com/pypi/packages/fa/e0/f2702df79f66d872532ce39227144264892afc75d28ce2e361145ddd79ca/types_protobuf-4.21.0.2-py3-none-any.whl (61 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 61.1/61.1 kB 1.4 MB/s eta 0:00:00
Collecting types-requests
  Downloading http://mirrors.tools.huawei.com/pypi/packages/fd/17/61e0d7203f3133aecead2d8b75f89d7b3c6664560e70d9030e69eb3c0d87/types_requests-2.28.11.5-py3-none-any.whl (14 kB)
Collecting types-setuptools
  Downloading http://mirrors.tools.huawei.com/pypi/packages/4d/72/abc24175dc3b249aa45f8e98bf4d1063503ae0044926c0710d700abdeca1/types_setuptools-65.6.0.2-py3-none-any.whl (48 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 49.0/49.0 kB 1.7 MB/s eta 0:00:00
Collecting types-six
  Downloading http://mirrors.tools.huawei.com/pypi/packages/38/32/77c09b1aa554cf1dbf23a44a9514d4f8db3ed51f003d4d97c628d994238e/types_six-1.16.21.4-py3-none-any.whl (14 kB)
Collecting types-urllib3
  Downloading http://mirrors.tools.huawei.com/pypi/packages/ab/35/7b813668050bfe0820ad84df0ba668adb6abb41ca2bb0f16ad4deabca279/types_urllib3-1.26.25.4-py3-none-any.whl (14 kB)
Installing collected packages: types-urllib3, types-six, types-setuptools, types-PyYAML, types-protobuf, types-Pillow, types-google-cloud-ndb, types-requests
Successfully installed types-Pillow-9.3.0.4 types-PyYAML-6.0.12.2 types-google-cloud-ndb-2.0.0.0 types-protobuf-4.21.0.2 types-requests-2.28.11.5 types-setuptools-65.6.0.2 types-six-1.16.21.4 types-urllib3-1.26.25.4

Generated Cobertura report: /home/stage/root/spack-stage-py-wandb-0.13.6-pjmssa244wsn6ybym6tqvxdmegj7h6u6/spack-src/mypy-results/cobertura.xml
Generated HTML report (via XSLT): /home/stage/root/spack-stage-py-wandb-0.13.6-pjmssa244wsn6ybym6tqvxdmegj7h6u6/spack-src/mypy-results/index.html
Success: no issues found in 331 source files
______________________________________________________________________________________________ summary ______________________________________________________________________________________________
  mypy: commands succeeded
  congratulations :)
```
I will close the issuse.
